### PR TITLE
gallery settings panel tooltips

### PR DESCRIPTION
**Pull request by ShamaySapir**

### CODE REVIEW

File: packages/editor-common/web/src/Components/Checkbox.jsx [hunk 1]
```diff
@@ -79,15 +78,7 @@ export default class Checkbox extends React.Component {
...
-        {contentForInfoIcon && (
-          <Tooltip
-            shouldRebuildOnUpdate={() => true}
-            content={contentForInfoIcon}
-            theme={styles.theme}
-          >
-            <InfoIcon className={styles.checkbox_infoIcon} />
-          </Tooltip>
-        )}
+        <InfoIcon tooltipTextKey={tooltipTextKey} />
```
Comment (shaulgo) [inline packages/editor-common/web/src/Components/Checkbox.jsx:81]: does it need the translation func?

File: packages/editor-common/web/src/Components/InputWithLabel.jsx [hunk 2]
```diff
@@ -34,14 +36,17 @@ class InputWithLabel extends Component {
...
     if (label) {
       return (
-        <label htmlFor={id}>
-          <span className={styles.inputWithLabel_label}>{label}</span>
+        <div>
+          <label htmlFor={id} className={generalstyles.infoContainer}>
+            <span className={styles.inputWithLabel_label}>{label}</span>
+            <InfoIcon iconStyles={styles.infoIcon} tooltipTextKey={tooltipTextKey} />
+          </label>
           {this.renderInput()}
```
Comment (shaulgo) [inline packages/editor-common/web/src/Components/InputWithLabel.jsx:47]: why did you move this out from the label?

File: packages/editor-common/web/statics/styles/input-with-label.scss [hunk 3]
```diff
@@ -35,3 +35,11 @@
 .inputWithLabel_fullHeight {
   height: 100%;
 }
+
+.infoIcon {
```
Comment (shaulgo) [inline packages/editor-common/web/statics/styles/input-with-label.scss:39]: seems like almost duplicate from the general.scss

File: packages/editor-common/web/src/Components/Checkbox.jsx (hunk 1 above)
Comment (ShamaySapir) [reply packages/editor-common/web/src/Components/Checkbox.jsx:81]: no, the translation func is inside InfoIcon, so we only need to send the tooltip key

File: packages/editor-common/web/statics/styles/input-with-label.scss (hunk 3 above)
Comment (ShamaySapir) [reply packages/editor-common/web/statics/styles/input-with-label.scss:39]: you're right, but it doesn't look good with the padding-top: 2px that there is in the general.scss, I needed to avoid this style in the case of input with lable.

File: packages/editor-common/web/src/Components/InputWithLabel.jsx (hunk 2 above)
Comment (ShamaySapir) [reply packages/editor-common/web/src/Components/InputWithLabel.jsx:47]: I did it for style reasons, but I found a better way.
